### PR TITLE
Tradheli: explanation of manual autorotation setup

### PR DESCRIPTION
**Pull request by bnsgeyer**
PR that explains how to set up manual autoroation.

### CODE REVIEW

File: copter/source/docs/traditional-helicopter-rsc-setup.rst [hunk 1]
```diff
@@ -57,3 +57,24 @@ Turbine Engine Start
...
+Practice (Manual) Autorotation Setup
+====================================
+
+In versions 4.4 and later, the ability to conduct power re-engagement from an autorotation has been added.  This feature will work with ESC's using their built-in governor, the ardupilot throttle curve or the ardupilot built-in governor.  Three parameters were added to accommodate this feature.
+
+* :ref:`H_RSC_AROT_MN_EN <H_RSC_AROT_MN_EN>` - Enables manual autorotations (0 - Disable, 1 - Enable)
+* :ref:`H_RSC_AROT_ENG_T <H_RSC_AROT_ENG_T>` - Time to ramp throttle back to motorlock enabled value during power recovery in seconds
+* :ref:`H_RSC_AROT_IDLE <H_RSC_AROT_IDLE>` - Idle throttle value in percent used during autorotation.  This can be set to the value that signals the ESC to perform a fast spool-up when performing a power recovery.
+
+ESC built-in Governor
```
Comment (Hwurzburg) [inline copter/source/docs/traditional-helicopter-rsc-setup.rst:70]: we should call this ESC Governor    you use "built-in" lower down and its confusing

File: copter/source/docs/traditional-helicopter-rsc-setup.rst [hunk 2]
```diff
@@ -57,3 +57,24 @@ Turbine Engine Start
...
+---------------------
+
+Enable manual autoratation by setting :ref:`H_RSC_AROT_MN_EN <H_RSC_AROT_MN_EN>` to 1.  Leave :ref:`H_RSC_AROT_ENG_T <H_RSC_AROT_ENG_T>` at the default setting of 1 second to quickly signal the ESC for fast spool-up.  Set the throttle idle value, :ref:`H_RSC_AROT_IDLE <H_RSC_AROT_IDLE>`, to the percent of throttle given by the ESC manual that signals the ESC to perform a fast spool-up.
+
+Ardupilot Throttle Curve or Built-in Governor
```
Comment (Hwurzburg) [inline copter/source/docs/traditional-helicopter-rsc-setup.rst:75]:
```suggestion
Ardupilot's Throttle Curve or Built-in Governor
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
We need to add instructions on actually how to initiate and bail out an autorotation...throttle interlock?

**Comment by Hwurzburg** [comment]
@bnsgeyer would you like me to fix these?  what about Heli Yaw Comp? you could explain it to me verbally and I can write it up if you wish....4.4 is about to happen I think

**Comment by bnsgeyer** [comment]
@Hwurzburg Yes, that would be great if you would incorporate your changes.  Thank You!!  I will take a stab at the Collective compensation hopefully tomorrow night.

**Comment by Hwurzburg** [comment]
@bnsgeyer did fixes...I think Autorotate is an important enough feature it deserved its own page so that it can appear in the side menu....so moved the additions to their own page and reference it in RSC setup

File: copter/source/docs/traditional-helicopter-rsc-setup.rst [hunk 3]
```diff
@@ -57,3 +57,10 @@ Turbine Engine Start
 ====================
 
 A special RC Auxiliary Function ("159") is implemented to provide the start signal RC switch for turbine engines. When armed and RSC is idle, the high position signals the helicopter rotor speed governor to ramp the throttle to full and back to idle, which signals the turbine engine ECU to initiate the start sequence. The switch must be set back low and aircraft has to be disarmed to re-enable this feature.
+
+Practice (Manual) Autorotation Setup
+====================================
+
+In versions 4.4 and later, the ability to conduct power re-engagement from an autorotation has been added.  This feature will work with ESC's using their built-in governor, the ardupilot throttle curve or the ardupilot built-in governor.  Three parameters were added to accommodate this feature.
+
```
Comment (bnsgeyer) [inline copter/source/docs/traditional-helicopter-rsc-setup.rst:65]: Probably want to remove the last sentence and be consistent with referencing the ESC governor.

Comment (Hwurzburg) [reply copter/source/docs/traditional-helicopter-rsc-setup.rst:65]: yeah and need to add links...thanks

**Review by Hwurzburg — APPROVED** [review]

**Pull request merged**